spec/requests/api/v2/providers/suggest_provider_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
+
+describe 'GET /suggest' do
+  let(:jsonapi_renderer) { JSONAPI::Serializable::Renderer.new }
+  let(:organisation) { create(:organisation) }
+  let(:provider1) { create(:provider, organisations: [organisation]) }
+  let(:provider2) { create(:provider, organisations: [organisation]) }
+  let(:next_recruitment_cycle) { find_or_create(:recruitment_cycle, :next) }
+  let(:provider3) { create(:provider, provider_name: provider2.provider_name, organisations: [organisation], recruitment_cycle: next_recruitment_cycle) }
+  let(:unauthorized_provider) { create(:provider, organisations: [create(:organisation)]) }
```

Comment:
what is that one for?

Reply:
That was for making sure we weren't spitting out providers the user wasn't in

Reply:
@tvararu is that right?

Reply:
I think it should be all providers actually, not just the ones they have access to

Reply:
^ confirmed by Paul H, it's all the accredited bodies in the system